shepherd/lowcar/flash.sh

```diff
@@ -0,0 +1,348 @@
+#!/bin/bash
```

Comment:
check with runtime if there is a new version of this, also we should totally use a module for this lol

Reply:
runtime said its probably fine for us to use old version of lowcar, @alexwongapps @akshitdewan thoughts?